types/v1/util.go

```diff
@@ -924,15 +924,25 @@ func MakeOrDefJivaReplicaArgs(vol *Volume, clusterIP string) []string {
 
 	//storSize := GetPVPStorageSize(profileMap)
 	storSize := vol.Capacity
+	cloneIP := vol.CloneIP
 
 	repArgs := make([]string, len(JivaReplicaArgs))
+	if cloneIP == "" {
```

Comment:
I believe its best to avoid this `if else` by structuring the code better.
Probably a new struct that deals with clone would have been great.
When the complexity arises with new business logic/features, a well defined struct will be able to handle this better.